dvc/remote/gdrive/__init__.py

```diff
@@ -37,25 +30,28 @@ def __init__(self, repo, config):
         self.path_info = self.path_cls(config[Config.SECTION_REMOTE_URL])
         self.root_content_cached = False
         self.root_dirs_list = {}
-        self.init_gdrive()
-
-    def init_gdrive(self):
-        self.gdrive = self.drive()
+        self._gdrive = None
         self.cache_root_content()
 
+    @property
```

Comment:
How about just using `cached_property` here, same as we do in other Remotes? You won't need `_gdrive` then :slightly_smiling_face:

Reply:
sounds good, thanks!